samples/Exporters/Web/Controllers/WeatherForecastController.cs

```diff
@@ -14,9 +16,12 @@ public class WeatherForecastController : ControllerBase
             "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
         };
 
+        private static HttpClient httpClient = new HttpClient();
+
         [HttpGet]
         public IEnumerable<WeatherForecast> Get()
         {
+            var res = httpClient.GetStringAsync("http://google.com").Result;

```

Comment:
I assume this is only for tests and not part of the final merge.

Reply:
Yes adding a comment.